s3_project/tests/test_phone_number.py

```diff
@@ -0,0 +1,12 @@
+from s3_project.classes.ExtractionClass import ExtractFromS3
+from s3_project.classes.talent_csv_cleaning import TalentCsv
+
+test_number = TalentCsv()
+
+def test_phone_number():

```

Comment:
No test that checks what happens if the phone number starts with a 0; what happens if this is the case?